public/index.html

```diff
@@ -36,15 +36,16 @@
   />
   <link href="css/styles.css" rel="stylesheet" type="text/css" />
 
-  <script src="js/lib/d3.min.js"></script>
+  <script src="https://d3js.org/d3.v7.min.js"></script>
   <script src="js/lib/topojson.v0.min.js"></script>
   <script src="js/lib/pathseg.js"></script>
 
-  <script src="https://api.tiles.mapbox.com/mapbox.js/v0.6.7/mapbox.js"></script>
+  <script src="https://api.mapbox.com/mapbox-gl-js/v2.3.1/mapbox-gl.js"></script>
   <link
-    href="https://api.tiles.mapbox.com/mapbox.js/v0.6.7/mapbox.css"
+    href="https://api.mapbox.com/mapbox-gl-js/v2.3.1/mapbox-gl.css"
     rel="stylesheet"
   />
```

Comment:
If the exact version is hardcoded anyway, we could also load this locally.

Reply:
Also changed to be local. If we bring on more modern javascript tooling like webpack/esbuild/etc, we might want to pull this in via the js build process instead of just using 'curl -O' to save it locally and check it into git, which is what I did.

Reply:
Yeah, let's leave this to future-@epaulson and future-@fnogatz :D